javalin-plugins/javalin-swagger-plugin/src/main/kotlin/io/javalin/openapi/plugin/swagger/SwaggerHandler.kt

```diff
@@ -16,7 +16,9 @@ class SwaggerHandler(
     private val routingPath: String,
     private val basePath: String?,
     private val tagsSorter: String,
-    private val operationsSorter: String
+    private val operationsSorter: String,
+    private val customStylesheetFiles: ArrayList<String>,
+    private val customJavaScriptFiles: ArrayList<String>
```

Comment:
It'd be better to use `List`, `ArrayList` is mutable by default and we don't need this functionality here.

Reply:
Good point - it doesn't matter what type of list is used here and immutability is a plus